src/AddOns/Bassloud/Bassloud.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace ManagedBass.Loud
+{
+    public static class BassLoud
+    {
+        const string DllName = "bassloud";
+
+        #region Version
+        [DllImport(DllName)]
+        static extern int GetVersion();
+
+        /// <summary>
+        /// Gets the Version of BassFx that is loaded.
+        /// </summary>
+        public static Version Version => Extensions.GetVersion(GetVersion());
+        #endregion
+
+        /// <summary>
+        /// Retrieves the channel that a loudness measurement is set on.
+        /// </summary>
+        /// <param name="Handle">The loudness measurement.</param>
+        /// <returns>If successful, the loudness measurement's channel handle is returned, else 0 is returned. Use <see cref="Bass.LastError" /> to get the error code.</returns>
+        /// <exception cref="Errors.Handle"><paramref name="Handle"/> is not valid.</exception>
+        [DllImport(DllName, EntryPoint = "BASS_Loudness_GetChannel")]
+        public static extern int GetChannel(int Handle);
+
+        /// <summary>
+        /// Retrieves the level of a loudness measurement.
+        /// </summary>
+        /// <param name="Handle">The loudness measurement handle.</param>
+        /// <param name="Mode">The measurement type to retrieve. One of the following. BassFlags.BassLoudnessCurrent, BassFlags.BassLoudnessIntegrated, BassFlags.BassLoudnessRange, BassFlags.BassLoudnessPeak, BassFlags.BassLoudnessTruePeak.</param>
+        /// <param name="Level">Pointer to a variable to receive the measurement level.</param>
+        /// <returns>If successful, TRUE is returned, else FALSE is returned. Use <see cref="Bass.LastError"/> to get the error code.</returns>
+        /// <exception cref="Errors.Handle"><paramref name="Handle"/> is not valid.</exception>
+        /// <exception cref="Errors.Parameter"><paramref name="Mode"/> is not valid. If requesting a duration with BASS_LOUDNESS_CURRENT then it exceeds what has been enabled.</exception>
+        /// <exception cref="Errors.NotAvailable">The requested measurement has not been enabled.</exception>
+        [DllImport(DllName, EntryPoint = "BASS_Loudness_GetLevel")]
+        public static extern bool GetLevel(int Handle, BassFlags Mode, ref float Level);
```

Comment:
```suggestion
        public static extern bool GetLevel(int Handle, BassFlags Mode, out float Level);
```

`out` may be better since current value of `Level` doesn't get used.

Reply:
I tried out before I tried ref, it caused a bunch of errors for some reason. My guess was the way pointers and stuff are done in C. I guess I'll give it another try later on though.

Reply:
I think this is fine for now if you can't get it to work, though I think `ref` and `out` should behave similarly in this case.

Reply:
`out` worked fine in my environment, but anyway I'll just revert it in my PR since it's just a minimal change.